tests/ut/ir/operators/test_tile_ops.py

```
@@ -2543,5 +2543,42 @@ def test_tile_add_symbolic_shape_mismatch_shows_var_names(self):
+    def test_tile_ci_rejects_uint_dtype(self):
+        with pytest.raises(ValueError, match=r"INT16.*INT32"):
+            tile.ci(0, [2, 16], dtype=DataType.UINT32)
+
+    def test_tile_ci_rejects_cols_equal_one(self):
+        with pytest.raises(ValueError, match="innermost dimension"):
+            tile.ci(0, [32, 1], dtype=DataType.INT32)
+
+    def test_tile_ci_rejects_start_dtype_mismatch(self):
+        span = ir.Span.unknown()
+        start = ir.Var("s", ir.ScalarType(DataType.INT16), span)
+        with pytest.raises(ValueError, match="start.*dtype"):
```

Ruff (0.15.10)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)